### PR TITLE
ribocode commit

### PR DESCRIPTION
**Pull request by xzt41**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by xzt41** [comment]
pls add my ribocode package, thanks!

File: recipes/ribocode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz
+  md5: e05ef843da351c4588e02bad3d5c3d65
+
+requirements:
+  build:
+    - python >2.6.6,<3
```
Comment (bgruening) [inline recipes/ribocode/meta.yaml]: Please skip python 3 builds in the top build section.

File: recipes/ribocode/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
+
+# Add more build steps here, if they are necessary.
```
Comment (bgruening) [inline recipes/ribocode/build.sh]: The comments can be removed here.

File: recipes/ribocode/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    - plot_orf_density --help
+    - ORF_count --help
+
+build:
+  entry_points:
+    - RiboCode=RiboCode.RiboCode:main
+    - prepare_transcripts=RiboCode.prepare_transcripts:main
+    - metaplots=RiboCode.metaplots:main
+    - plot_orf_density=RiboCode.plot_orf_density:main
+    - ORFcount=RiboCode.RPF_count_ORF:
```
Comment (bgruening) [inline recipes/ribocode/meta.yaml]: here is something missing, maybe the main() function?

**Comment by xzt41** [comment]
thank you so much. I have modified my files according your suggestion. Please merge my branch. And I don't what happened in travis-ci, it is hard to understand the error in log file.

File: recipes/ribocode/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz
+  md5: e05ef843da351c4588e02bad3d5c3d65
+
+requirements:
+  build:
+    - python # [py27]
```
Comment (bgruening) [inline recipes/ribocode/meta.yaml]: You should include something like that:
```
build:
  skip: true  # [not py27]
```

Please have a look at other packages or our guidelines.

File: recipes/ribocode/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+build:
+  skip: True # [not py27]
+  entry_points:
+    - RiboCode=RiboCode.RiboCode:main
+    - prepare_transcripts=RiboCode.prepare_transcripts:main
+    - metaplots=RiboCode.metaplots:main
+    - plot_orf_density=RiboCode.plot_orf_density:main
+    - ORFcount=RiboCode.RPF_count_ORF:main
+  script:
+    python setup.py install
```
Comment (bgruening) [inline recipes/ribocode/meta.yaml]: as you have this line here included you can remove this build.sh file completely ...

**Pull request closed without merging**